File: example_app/src/main/java/com/jampez/uceh_example/MyApplication.java
/*
 *
 *  * Copyright © 2018 Rohit Sahebrao Surwase.
 *  *
 *  *    Licensed under the Apache License, Version 2.0 (the "License");
 *  *    you may not use this file except in compliance with the License.
 *  *    You may obtain a copy of the License at
 *  *
 *  *        http://www.apache.org/licenses/LICENSE-2.0
 *  *
 *  *    Unless required by applicable law or agreed to in writing, software
 *  *    distributed under the License is distributed on an "AS IS" BASIS,
 *  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  *    See the License for the specific language governing permissions and
 *  * limitations under the License.
 *
 */
package com.jampez.uceh_example;

import android.app.Application;

import com.jampez.uceh.UCEHandler;

/**
 * <b></b>
 * <p>This class is used to </p>
 * Created by Rohit.
 */
public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        //Initialize UCE Handler library
        new UCEHandler.Builder(getApplicationContext())
                .setTrackActivitiesEnabled(true)
                .setBackgroundModeEnabled(true)
                .build();
    }
}

File: example_app/src/main/java/com/jampez/uceh_example/MyApplication.kt
package com.jampez.uceh_example

import android.app.Application
import com.jampez.uceh.features.bitbucket.BitBucket
import com.jampez.uceh.features.github.Github
import com.jampez.uceh.features.gitlab.GitLab
import com.jampez.uceh.features.uce.UCEHandler
import com.jampez.uceh.utils.Mode

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        //Initialize UCE Handler library
        val uceHandlerBuilder = UCEHandler.Builder(applicationContext)
        uceHandlerBuilder.setTrackActivitiesEnabled(true)
        uceHandlerBuilder.setIconDrawable(R.mipmap.ic_launcher)
        uceHandlerBuilder.setBackgroundModeEnabled(true)
        uceHandlerBuilder.setCanViewErrorLog(false)
        uceHandlerBuilder.setCanShareErrorLog(false)
        uceHandlerBuilder.setIssueMode(Mode.Manual)
        uceHandlerBuilder.setIssueButtonText("Create a Support Ticket")
        uceHandlerBuilder.setGithubService(Github.Builder()
                .setAccessToken("<< github access token >>")
                .setRepoName("<< repo name >>")
                .setUsername("<< username >>")
        )
        uceHandlerBuilder.setBitBucketService(BitBucket.Builder()
                .setUsername("<< username >>")
                .setAppPassword("<< app password >>")
                .setProjectName("<< project name >>")
                .setRepoName("<< repo name >>")
        )
        uceHandlerBuilder.setGitLabService(GitLab.Builder()
                .setAccessToken("<< access token >>")
                .setProjectID("<< project id >>")
        )
        uceHandlerBuilder.build()
    }
}